corehq/couchapps/attachments/views/attachments/map.js

```diff
@@ -10,6 +10,13 @@ function (doc) {
                     attachments[key] = doc._attachments[key];
                 }
             }
+            for (var key in doc.external_blobs) {
+                if (doc.external_blobs.hasOwnProperty(key) &&
+                    doc.external_blobs[key].content_type !== "text/xml") {
+                    media += 1;
+                    attachments[key] = doc.external_blobs[key];
+                }
+            }
```

Comment:
I assume this is necessary to keep the view functioning properly, but I haven't made many changes like this before. It seems a shame to have kick off a huge reindex for this since when it goes live the index will be up to date and only new `XFormInstance` docs created after the merge will need this new index code. Is there a way to avoid the reindex in this type of situation?